package.json

```diff
@@ -41,7 +41,8 @@
     "node": ">=14.13.1"
   },
   "dependencies": {
-    "ts-morph": "^13.0.2",
+    "deno.ns": "^0.8.0",
```

Comment:
`deno.ns` is deprecated in favor of https://www.npmjs.com/package/@deno/shim-deno

Reply:
Alright, I'll change it.